plugin.xml

```diff
@@ -79,7 +72,7 @@
         </config-file>
 
         <config-file target="*-Info.plist" parent="ApptentivePluginVersion">
-            <string>4.0.1</string>
+            <string>4.0.2</string>
```

Comment:
@frankus Do we still need to do this? This is originally (I think) part of the change I made where I reformatted the Info.plist file to be XML instead of binary, and then started saving the distribution version in here. I remember you saying that you and Wooster thought that was a bad idea for some reason, but don't remember if you implemented a different solution.